gazebo_ros_control/CMakeLists.txt

```diff
@@ -31,9 +33,11 @@ catkin_package(
     joint_limits_interface
     urdf
     angles
-  INCLUDE_DIRS include
-  LIBRARIES ${PROJECT_NAME} default_robot_hw_sim
-  DEPENDS gazebo
```

Comment:
I read that documentation and it does not clarify anything for me. It just says ``DEPENDS - Non-catkin CMake projects that this project depends on`` but then does it also not include things like Boost?

Another example - why does ``gazebo_ros`` not [DEPEND on GAZEBO](https://github.com/ros-simulation/gazebo_ros_pkgs/blob/kinetic-devel/gazebo_ros/CMakeLists.txt#L55)? My change in this PR was inspired by that CMakeLists file.